modules/@apostrophecms/image-widget/views/widget.html

```diff
@@ -1,31 +1,39 @@
-{% if data.options.className %}
-  {% set className = data.options.className %}
-{% elif data.manager.options.className %}
-  {% set className = data.manager.options.className %}
-{% endif %}
+{% if apos.util.eqStrict(data.widget.aposPlaceholder, true) and data.manager.options.placeholderUrl %}
+  <img
+    src="{{ apos.asset.url(data.manager.options.placeholderUrl) }}"
+    alt="image placeholder"
+    width="100%"
```

Comment:
Should have a class that can be styled. We should supply default styles to the extent we supply them for the regular content of the image widget.

Reply:
Using the width attribute is not appropriate because it more or less forces them to override all of widget.html if they want to do something else.